master/static/migrations/20230918202017_add-allocation-accelerators-table.tx.up.sql

```diff
@@ -0,0 +1,7 @@
+ CREATE TABLE public.allocation_accelerators (
+    container_id text NOT NULL PRIMARY KEY,
+    allocation_id text NOT NULL REFERENCES public.allocations(allocation_id),
```

Comment:
add a ON DELETE cascade to this allocation_id

Reply:
Done!